routes/professions.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express');
+const pool = require('../connection/connection');
+
+const router = express.Router();
+
+// Get Method
+router.get('/:id', (req, res) => {
```

Comment:
Make sure to add another GET method with just `/` so we can display all the rows if needed. Same with other files 😄